### DISCUSSION TITLE
404 with _app getInitialProps

### DISCUSSION THREAD

**Discussion by 1derwood** [vercel/next.js#11263] [Help] [answered]
This seems to be a new fatal error in the latest version?

`pages/404` can not have getInitialProps/getServerSideProps, https://err.sh/zeit/next.js/404-get-initial-props

If you have getInitialProps in _app, is there a way to have a functioning 404 page?

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like you're running into this issue https://github.com/zeit/next.js/issues/10919 which was resolved in `v9.3.1` of Next.js specifically this PR https://github.com/zeit/next.js/pull/10974. You are allowed to use `pages/404` even if you have `getInitialProps` in `pages/_app` although it will not be statically optimized currently since you are defining an app wide data dependency when using `getInitialProps` in `pages/_app`
